client/components/Assignments/AssignmentEditor.tsx

```diff
@@ -297,15 +298,18 @@ export class AssignmentEditorComponent extends React.Component {
                         />
                     </Row>
                 ) : (
-                    <SelectUserInput
```

Comment:
Besides this, only one usage remains of this component. Can you replace it too and remove this component completelly?

Reply:
I saw you removed usages, but also go ahead and remove the component itself.